### PR TITLE
Update imager

### PR DESCRIPTION
**Pull request by mirceacaprioru**
This patch series adds support for configuring Zynq and ZynqMP project for the ADI Kuiper Linux. The user has the option to select the platform and the project that he wants to be configured after the image has been written on the  SD card. 

This feature does not use the hidden advanced menu (CTRL-SHIFT-X) but has two additional combo boxes added to the base GUI. This can be enhanced in later iterations, and also added functionality for Altera/Intel projects along with preloader update on the SD card.  

image

By the way I create pretty ugly GUIs so I apologize in advance.

### CODE REVIEW

File: config.h [hunk 1]
```diff
@@ -8,7 +8,10 @@
 
 
 /* Repository URL */
-#define OSLIST_URL                        "https://raw.githubusercontent.com/analogdevicesinc/adi-kuiper-imager/update_imager/os_list.json"
+#define OSLIST_URL                        "https://raw.githubusercontent.com/analogdevicesinc/adi-kuiper-imager/qml/os_list.json"
+
```
Comment (nunojsa) [inline config.h:12]: Should these links be valid? It looks like they are not... Or will we add them in the future?

**Review by nunojsa — APPROVED** [review]
LGTM

File: config.h (hunk 1 above)
Comment (mirceacaprioru) [reply config.h:12]: will become valid once we merge this.

**Comment by mirceacaprioru** [comment]
Changelo v2
-remove QML debug prints

**Review by adisuciu — APPROVED** [review]

**Pull request merged**